### ISSUE TITLE
High connection latency

### ISSUE THREAD

**Issue by insideable** [aio-libs/aiohttp#448]
If you try to run aiohttp benchmark with concurrency=1, you'll get strange results:

```
Results for run_aiohttp
RPS: 15,        mean: 67.338 ms, standard deviation 56.935 ms    median 43.476 ms
Results for run_tornado
RPS: 820,       mean:  1.216 ms, standard deviation 0.395 ms     median 1.154 ms
Results for run_twisted
RPS: 610,       mean:  1.678 ms, standard deviation 0.638 ms     median 1.399 ms
```

I get similar results with `wrk -c1`: aiohttp has big connection latency.

Note: this issue is not related to asyncio itself (simple HTTP server implementation shows  ~ 1ms per request)

**Comment by carlcarl** [comment]
I used python3.5 to test this several times without any significant problem(`python3 async.py -c 1`).

Results for run_aiohttp
RPS: 675,   mean: 1.418 ms, standard deviation 0.402 ms median 1.419 ms
Results for run_tornado
RPS: 681,   mean: 1.410 ms, standard deviation 0.290 ms median 1.406 ms
Results for run_twisted
RPS: 906,   mean: 1.051 ms, standard deviation 0.314 ms median 0.999 ms

I think maybe this can be closed(?)

**Comment by asvetlov** [comment]
No, I still see the issue

**Comment by asvetlov** [comment]
Fixed by #680 #685
